apps/test/netsim/NetSimRouterNode.js

```diff
@@ -172,14 +187,42 @@ describe("NetSimRouterNode", function () {
    * Call tick() with an advanced time parameter on the argument until
    * no change is detected in the log table.
    * @param {NetSimRouterNode|NetSimLocalClientNode} tickable
+   * @param {number} [startTime] default 1
+   * @param {number} [timeStep] default 1
+   * @returns {number} Next tick time after stabilized
    */
-  var tickUntilLogsStabilize = function (tickable) {
-    var t = 1, lastLogCount;
+  var tickUntilLogsStabilize = function (tickable, startTime, timeStep) {
+    var t = utils.valueOr(startTime, 1);
+    timeStep = utils.valueOr(timeStep, 1);
+    var lastLogCount;
     do {
       lastLogCount = countRows('logTable');
       tickable.tick({time: t});
-      t += 1;
+      t += timeStep;
     } while (countRows('logTable') !== lastLogCount);
+    return t;
+  };
+
+  /**
+   * Build a fake message log that can be passed into
+   * NetSimLocalClientNode.initializeSimulation as the sent or received log,
+   * and can be used to sense whether a particular message would be passed
+   * to the sent/received log UI components.
+   * @returns {{log: Function, getLatest: Function}}
+   */
+  var makeSensingMessageLog = function () {
```

Comment:
There are well-defined names for various types of objects used for testing: http://en.wikipedia.org/wiki/Test_double

This looks like a fake, so let's call it makeFakeMessageLog.